src/NinjaTools.FluentMockServer.API/Proxy/Evaluation/Models/EvaluationResultBase.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace NinjaTools.FluentMockServer.API.Proxy.Evaluation.Models
+{
+    public abstract class EvaluationResultBase : IEvaluationResult
+    {
+        public EvaluationResultBase([NotNull] EvaluationContext context)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of this constructor to 'protected'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)